storops/unity/resource/pool.py

```diff
@@ -203,6 +206,51 @@ def disk_groups(self):
                 dgs[pd.disk_group.get_id()] = [pd]
         return dgs
 
+    @version('<5.1')
+    def is_compression_enabled(self):
```

Comment:
Change `is_compression_enabled` to `is_compression_supported`? Because cannot enable/disable compression on pool.

Reply:
Done.